### PR TITLE
Add otrs db user

### PR DESCRIPTION
**Pull request by danypr92**
We need a user to be used by OTRS to read the party view to you as
Customers in OTRS. By this reason we add a user in PostgreSQL `otrs`
with the privilege of read the view `party_otrs_view`.

Also I move the vars related with OTRS to the OTRS vars block and add the pg_hba entries needed to access with the OTRS user

### CODE REVIEW

**Comment by enricostano** [comment]
How were we doing this until now? OTRS already has access to a PostgreSQL view right?

**Comment by danypr92** [comment]
@enricostano yes, but we manage the creation of the user, the privileges and the view with a script in the Eticom Module. And we modified the hba entries manually...

File: inventory/host_vars/stagingtryton.coopdevs.org/config.yml [hunk 1]
```diff
@@ -32,20 +32,49 @@ nginx_config: 'uwsgi-dev.nginx'
...
+          6564616465653237610a633638323034643832646133316463356132363737326534323362323864
+          6261
+otrs_new_ip: !vault |
+          $ANSIBLE_VAULT;1.1;AES256
+          32376231626565356166623434343731313732376137366432333234633565616630316634376437
+          3339313766303738306135643633336331393834326536380a373631306239303033653261336363
+          33316333643938626634313635343262303261656339383833633430323631643131613331353830
+          3930376163643638390a646363383936366137306665313966373132646434616236356434373430
+          3862
+
```
Comment (sauloperez) [inline inventory/host_vars/stagingtryton.coopdevs.org/config.yml]: I see we would really benefit from splitting this file following its comments: otrs.yml and open_cell.yml, at least. We get it for free with Ansible :tada: !

File: playbooks/provision.yml [hunk 2]
```diff
@@ -19,6 +19,10 @@
           - name: '{{ db_user }}'
             password: '{{ db_user_password }}'
             role_attr_flags: CREATEDB
+          - name: '{{ otrs_db_user }}'
+            password: '{{ otrs_db_password }}'
+            priv: party_otrs_view:SELECT
```
Comment (sauloperez) [inline playbooks/provision.yml]: I don't mean to do it here but as I'm already working on creating another view for OTRS, we should think on a different and more general way to grant permissions to OTRS IMO, so we don't have to change this file every time new requirements come in :moneybag: .

**Pull request merged**